package.json

```diff
@@ -28,15 +28,15 @@
     "build:web": "vite build",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
-    "icons": "cd resources && npx png2icons ../static/favicon.png icon -allwe -bc"
+    "icons": "cd resources && npx png2icons ../static/favicon.png icon -allwe -bc",
+    "postinstall": "npx base-volta-off-of-nwjs@latest"
   },
   "type": "module",
   "dependencies": {
     "node-static": "^0.7.11",
     "portfinder": "^1.0.32"
   },
   "volta": {
-    "node": "20.5.0",
-    "npm": "9.8.0"
+    "node": "20.7.0"
```

Comment:
Since the node value is now auto-updated based on NW.js, I removed the `npm` value so it will default to the version of npm that ships with that Node version.